CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # tartiflette-plugin-scalars
```

Comment:
Root level changelog should only contain a table like : https://github.com/tartiflette/tartiflette/blob/master/CHANGELOG.md